### PR TITLE
Made error message format consistent

### LINKED ISSUE #590: Inconsistent error message formatting
Some error messages show up as `🚫 ope: error` while others show as `🚫 error`

### PR DESCRIPTION
**Pull request by therithwikrayani**
Closes #590

Made error message format consistent

- [x] I updated the changelog

### CODE REVIEW

File: src/utils/error-msg.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default (error?: string | Error): string => {
     if (typeof error === 'string') {
       str = `🚫 ${error}`;
     } else if (error instanceof Error) {
-      str = `🚫 ope: ${error.message}`;
+      str = `🚫 ${error.message}`;
```
Comment (codetheweb) [inline src/utils/error-msg.ts:8]:
```suggestion
      str = `🚫 ope: ${error.message}`;
```

File: src/utils/error-msg.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export default (error?: string | Error): string => {
     if (typeof error === 'string') {
       str = `🚫 ${error}`;
```
Comment (codetheweb) [inline src/utils/error-msg.ts:6]:
```suggestion
      str = `🚫 ope: ${error}`;
```

**Review by codetheweb — CHANGES_REQUESTED** [review]
I'd like to keep the `ope` prefix. :)

Are there any other spots to your knowledge that would have to be changed too?

**Comment by therithwikrayani** [comment]
Very well, done as suggested

**Comment by therithwikrayani** [comment]
No errors are thrown without using `error-msg.ts`, so shouldn't have to make changes anywhere else.

**Comment by codetheweb** [comment]
Thanks!

**Pull request merged**